build/ribasim_cli/tests/test_models.py

```diff
@@ -7,13 +7,13 @@
 
 
 @pytest.mark.parametrize(
-    "model_name,model_constructor",
-    ribasim_testmodels.constructors.items(),
+    "model_constructor",
+    ribasim_testmodels.constructors.values(),
 )
-def test_ribasim_cli(model_name, model_constructor, tmp_path):
+def test_ribasim_cli(model_constructor, tmp_path):
     model = model_constructor()
     assert isinstance(model, ribasim.Model)
-    model.write(tmp_path / model_name)
+    model.write(tmp_path)
```

Comment:
With this the model folder name becomes a random `tmp_path` instead of the model name. That makes it hard to find the right model for manual inspection.

Reply:
`tmp_path` isn't random.
It gets the name according to the test parameters

Reply:
Would you be able to identify the model from the test parameters?

Reply:
It does on my machine.
But please, try it out yourself :)

Reply:
Nice. I couldn't afford to download new binaries over 4G to test 😓